### PR TITLE
API changes for array migration

### PR DESCRIPTION
**Pull request by nidtara**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/267 |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by delldubey — APPROVED** [review]
lgtm

File: api/v1alpha1/dellcsimigrationgroup_types.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
+ Copyright © 2021 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (shanmydell) [inline api/v1alpha1/dellcsimigrationgroup_types.go:2]: Hi @nidtara , please change the year

File: api/v1alpha1/dellcsimigrationgroup_types.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+// DellCSIMigrationGroup is the Schema for the dellcsimigrationgroup API
+type DellCSIMigrationGroup struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              DellCSIMigrationGroupSpec   `json:"spec,omitempty"`
+	Status            DellCSIMigrationGroupStatus `json:"status,omitempty"`
+}
+
+// +kubebuilder:object:root=true
```
Comment (shanmydell) [inline api/v1alpha1/dellcsimigrationgroup_types.go:51]: shall we remove this comment?

File: api/v1alpha1/groupversion_info.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*
+ Copyright © 2022 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (shanmydell) [inline api/v1alpha1/groupversion_info.go:2]: change the year here

File: api/v1alpha1/dellcsimigrationgroup_types.go (hunk 2 above)
Comment (nidtara) [reply api/v1alpha1/dellcsimigrationgroup_types.go:51]: The is for controller-gen automation. This is the format.

File: api/v1alpha1/dellcsimigrationgroup_types.go (hunk 1 above)
Comment (nidtara) [reply api/v1alpha1/dellcsimigrationgroup_types.go:2]: will update this.

**Review by shanmydell — APPROVED** [review]

**Review by delldubey — APPROVED** [review]
lgtm

**Pull request merged**